src/plan_partialize.c

```diff
@@ -70,13 +71,20 @@ check_for_partialize_function_call(Node *node, PartializeWalkerState *state)
 		if (state->looking_for_agg)
 		{
 			state->looking_for_agg = false;
-			aggref->aggsplit = AGGSPLIT_INITIAL_SERIAL;
-
-			if (aggref->aggtranstype == INTERNALOID &&
-				DO_AGGSPLIT_SERIALIZE(AGGSPLIT_INITIAL_SERIAL))
-				aggref->aggtype = BYTEAOID;
-			else
-				aggref->aggtype = aggref->aggtranstype;
+			if (state->fix_aggref == TS_FIX_AGGREF)
```

Comment:
I suggest to write a comment here, why it is needed and what it actually do

Reply:
@pmwkaa It was originally there, I just re-use function without doing modification. I am guessing what it does, but I am not so fluent in the PG planning to be sure. So I am not sure I should comment here.